File: 3rd Sem/OOP Lab/2022_07_26/q4.cpp
#include<iostream>
using namespace std;

//4.Write a program to find out the area of a circle and volume of a sphere by using function overloading concept. 
//(Use one function name as AREA-VOL)

float area_vol(int x);
float area_vol(float x);

int main(){
    int r1;
    float r2;

    cout<<"Enter radius of circle: ";
    cin>>r1;
    cout<<"Enter radus of sphere: ";
    cin>>r2;

    cout<<"Area of circle: "<<(float)area_vol(r1)<<endl;
    cout<<"Volume of sphere: "<<(float)area_vol(r2)<<endl;

    return 0;
}

float area_vol(int x){
    float res = 3.1416*x*x;
    return res;
}

float area_vol(float x){
    float res =  (4*3.1416*x*x*x)/3.0;
    return res;
}

File: 4th sem/WT Lab/2023_02_20/q4.java
public class q4 {
    public static void main(String[] args){
        class Shape{
            Shape(int r){
                System.out.println("Area of circle: " + 3.14*r*r);
            }
            Shape(int b, int h){
                System.out.println("Area of triangle: " + 0.5*b*h);
            }
            Shape(int l, float b){
                System.out.println("Area of rectangle: " + l*b);
            }
        }
        new Shape(5);
        new Shape(5, 6);
        new Shape(5, 6.0f);
    }   
}
